lib/ansible/modules/packaging/os/apt.py

```diff
@@ -124,6 +124,14 @@
     type: bool
     default: 'no'
     version_added: "2.1"
+  fail_on_autoremove:
+    description:
+      - 'Corresponds to the C(--no-remove) option for C(apt).'
+      - 'If C(yes), it is ensured that no packages will be removed or the task will fail.'
+      - 'C(fail_on_autoremove) is only supported with state except C(absent)'
+    type: bool
+    default: 'no'
+    version_added: "2.11"
```

Comment:
@samdoran Should I change this line to `version_added: "2.11"` to resolve CI failure?
(Also for https://github.com/ansible/ansible/pull/70387)